device.go

```diff
@@ -50,6 +50,7 @@ var (
 	getPCIDeviceName    = getPCIDeviceNameImpl
 	getDevicePCIAddress = getDevicePCIAddressImpl
 	scanSCSIBus         = scanSCSIBusImpl
+	rootfsNvdimmDisk    = "/dev/pmem0p1"
```

Comment:
I think there is a different way to the rootfs device

Reply:
@devimc PR updated. PTAL

Reply:
thanks